### ISSUE TITLE
python_targets_python3_6 is getting deprecated

### ISSUE THREAD

**Issue by fcolecumberri** [ros/ros-overlay#1010]
on `/usr/portage/profiles/base/use.mask`:

```
# Michał Górny <mgorny@gentoo.org> (2021-01-01)
# The support for python3.6 target in Gentoo is discontinued.
# The flags are masked to permit removal of the last backport packages.
# Once they're gone, the eclass support will be disabled and the flags
# removed.
python_targets_python3_6
python_single_target_python3_6
```
the `python_targets_python3_6` has been masked, I do not know if changing the whole overlay from `python3_6` to `python3_{7, 8, 9}` would be a problem.

**Comment by flabrosse** [comment]
And as of 01/06/2021 the default will be 3_9.  This is a race that is difficult to follow.

For info, I have a working ros-noetic with 3_8 and have submitted a pull request (#1013) to have that included.  I guess I now need to redo it to include 3_9.  It would be easier to do if the pull request  was pulled in.

**Comment by awesomebytes** [comment]
I don't have any feedback to help, but I'd like to say that I really admire
your push to keep this up to date. The amount of breaking changes on the
Python language are becoming rather annoying from a maintainer point of
view.

Again, you rock!

**Comment by flabrosse** [comment]
Yeah, this constant chase is making me wondering if I should not just change to ubuntu (now that there is a server version that would be more appropriate to our robots than a desktop version)...
